frontend/src/utils/testing/intlMocks.ts

```diff
@@ -42,4 +42,79 @@ export const mockMessages = {
       },
     },
   },
+  Vision: {
```

Comment:
since the useTranslations hook looks like it's being mocked, I'm not sure why we need this, but the tests do fail without it. Would be nice to rely on the mock, but totally not blocking!